packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -956,42 +953,6 @@ export const audiusBackend = ({
     }
   }
 
-  async function getSocialHandles(user: User) {
```

Comment:
Just to refresh my memory: We were just waiting on the production migration to run before removing this? And has that been confirmed run on all DNs?

Reply:
yeah was waiting on 0.6.132 and by now all DNs are on that version or a more recent one. also, this client code is getting deployed next week